app/views/errors/SignedYouOutView.scala.html

```diff
@@ -0,0 +1,36 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.ViewUtils._
+@import config.FrontendAppConfig
+
+@this(
+    govukLayout: templates.GovukLayoutWrapper,
+    h1: components.h1,
+    link: components.linkAsButton
+)
+
+@()(implicit request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
+
+@govukLayout(Some(titleNoForm("signed_out.title")), signOut = false) {
+
+    @h1(messages("signed_out.heading"))
+
+    <p class="govuk-body">@messages("signed_out.status")</p>
+
+    @link(appConfig.loginUrl, messages("timeout.signIn"), Some("linkButton") )
```

Comment:
it should be task list url for login

Reply:
This instruction goes against the requirement stated in the Jira ticket. We would need to discuss this with Blaine, and he should change the ticket.

Reply:
This is expected application behavior and its implicit requirement.  Url used here is not correct. I hope you know the application behaviour.

Reply:
Changed as described in comment.